sqlmesh/core/model/definition.py

```diff
@@ -113,6 +113,7 @@ class _Model(ModelMeta, frozen=True):
     python_env_: t.Optional[t.Dict[str, Executable]] = Field(default=None, alias="python_env")
     jinja_macros: JinjaMacroRegistry = JinjaMacroRegistry()
     mapping_schema: t.Dict[str, t.Any] = {}
+    inline_audits: t.Optional[t.Dict[str, ModelAudit]] = None
```

Comment:
should we make this a dict? and then you don't have to check for existence on line 472?

Reply:
Sure I can try this